### DISCUSSION TITLE
new commit for AnimateDiff....error

### DISCUSSION THREAD

**Discussion by djptr** [continue-revolution/sd-webui-animatediff#426] [Q&A] [answered]
There's a new commit for AnimateDiff. When I apply the update my console gives an error:

Saving backup of webui/extension state to C:\SD\stable-diffusion-webui\config_states\2024_02_09-13_16_48_Backup (pre-update).json.
*** Error getting updates for sd-webui-animatediff
Traceback (most recent call last):
File "C:\SD\stable-diffusion-webui\modules\ui_extensions.py", line 45, in apply_and_restart
ext.fetch_and_reset_hard()
File "C:\SD\stable-diffusion-webui\modules\[extensions.py](https://extensions.py/)", line 181, in fetch_and_reset_hard
repo.git.fetch(all=True)
File "C:\SD\stable-diffusion-webui\venv\lib\site-packages\git\[cmd.py](https://cmd.py/)", line 741, in <lambda>
return lambda *args, **kwargs: self._call_process(name, *args, **kwargs)
File "C:\SD\stable-diffusion-webui\venv\lib\site-packages\git\[cmd.py](https://cmd.py/)", line 1315, in _call_process
return self.execute(call, **exec_kwargs)
File "C:\SD\stable-diffusion-webui\venv\lib\site-packages\git\[cmd.py](https://cmd.py/)", line 1109, in execute
raise GitCommandError(redacted_command, status, stderr_value, stdout_value)
git.exc.GitCommandError: Cmd('git') failed due to: exit code(1)
cmdline: git fetch --all
stderr: 'error: cannot lock ref 'refs/remotes/origin/forge/dev': 'refs/remotes/origin/forge' exists; cannot create 'refs/remotes/origin/forge/dev'
From https://github.com/continue-revolution/sd-webui-animatediff
! [new branch] forge/dev -> origin/forge/dev (unable to update local ref)
error: cannot lock ref 'refs/remotes/origin/forge/master': 'refs/remotes/origin/forge' exists; cannot create 'refs/remotes/origin/forge/master'
! [new branch] forge/master -> origin/forge/master (unable to update local ref)
error: some local refs could not be updated; try running 'git remote prune origin' to remove any old, conflicting branches'
_______________

It says I should run 'git remote prune origin' to remove any old, conflicting branches.
Which directory should I run the git cmd from?  (Forgive my ignorance, but I dont want to break something...)

thank you.

**Comment by zappityzap** [comment, marked as answer]
C:\SD\stable-diffusion-webui\extensions\sd-webui-animatediff

**Reply by djptr** [reply]
Works! Thank you Zappityzap.
